docs/content/faq.md

```diff
@@ -144,5 +144,39 @@ The Azure mixins are going to be full fledged tools that communicate with the
 Azure APIs, and provide an improved user experience. It all depends on how much
 time you want to invest, and what you are starting from.
 
+## Does the Porter has its own Registry like DockerHub, ACR,Quay etc?
+
+ No, Porter does have its Registry, rather use an OCI compliant artifact store. And now DockerHub is also OCI compliant registry. The list of registries that works with CNAB ie bundles [here](https://porter.sh/compatible-registries/) .
+
+## What is the use of Mixins?
+
+Mixins provide the following features:
+ * Install a tool into the bundle. It handles editing your dockerfile for you.
+ * Mixins can adapt imperative command line tools to work with desired state. 
+   >For example, the uninstall command should be re-runnable. I should be able to re-run porter uninstall if there was an error and it     should pick up where it left off. Most command line tools are imperative though, and don’t handle errors in a way that works well with that (e.g. returning a failure when you attempt to delete something that   is already deleted).
```

Comment:
Using a blockquote here (the `>` formatting) causes this part to look like we are quoting something, when it is an example that supports the previous sentence. Let's keep this as a normal paragraph formatted text to improve readability.

```suggestion
 * Mixins can adapt imperative command line tools to work with desired state. 
 
    For example, the uninstall command should be re-runnable. I should be able to re-run porter uninstall if there was an error and it should pick up where it left off. Most command line tools are imperative though, and don’t handle errors in a way that works well with that (e.g. returning a failure when you attempt to delete something that   is already deleted).
```

Please make sure to make the same change to other places in the faq where blockquotes were added.